.rubocop_todo.yml

```diff
@@ -319,27 +313,34 @@ Style/Encoding:
   Exclude:
     - 'trollolo.gemspec'
 
-# Offense count: 7
+# Offense count: 8
 # Configuration parameters: MinBodyLength.
 Style/GuardClause:
   Exclude:
     - 'lib/burndown_chart.rb'
     - 'lib/card.rb'
     - 'lib/cli_settings.rb'
+    - 'lib/remote_burndown.rb'
     - 'lib/scrum/backlog_mover.rb'
     - 'lib/scrum_board.rb'
     - 'lib/settings.rb'
 
-# Offense count: 8
+# Offense count: 9
 # Cop supports --auto-correct.
 Style/IfUnlessModifier:
   Exclude:
     - 'lib/burndown_chart.rb'
     - 'lib/cli/burndown.rb'
+    - 'lib/remote_burndown.rb'
     - 'lib/scrum/sprint_cleaner.rb'
     - 'spec/integration/integration_spec_helper.rb'
     - 'spec/unit/spec_helper.rb'
 
+# Offense count: 1
+Style/MixinUsage:
+  Exclude:
+    - 'spec/unit/burndown_plot_spec.rb'
```

Comment:
some of the offenses may be because you move code around, but some other looks new. You shouldn't introduce new offenses... :eyes:

Reply:
Fixed!